### PR TITLE
added browserify build

### LINKED ISSUE #8: Add browser build and/or esmodule support
I see people using this in their browser

### PR DESCRIPTION
**Pull request by therealadityashankar**
closes #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,12 +5,14 @@
...
   "browser": "p2pt.js",
   "scripts": {
+    "build": "browserify ./p2pt.js -s P2PT > ./dist/p2pt.umd.js",
     "test-browser": "BROWSER_TEST=1 airtap --local test/*.js",
     "test-node": "node test/*.js",
     "test": "standard && npm run test-node && npm run test-browser"
   },
   "dependencies": {
     "bittorrent-tracker": "^9.14.5",
+    "browserify": "^16.5.1",
```
Comment (subins2000) [inline package.json:15]: Shouldn't this better be in devDependencies ?

File: package.json [hunk 2]
```diff
@@ -5,12 +5,14 @@
   "main": "node.js",
   "browser": "p2pt.js",
   "scripts": {
+    "build": "browserify ./p2pt.js -s P2PT > ./dist/p2pt.umd.js",
```
Comment (subins2000) [inline package.json:8]: Is it possible to also minify the build ? And change filename to `p2pt.min.js`

**Comment by therealadityashankar** [comment]
made the required changes @subins2000

File: package.json [hunk 3]
```diff
@@ -5,22 +5,26 @@
...
     "wrtc": "^0.4.6"
   },
   "devDependencies": {
     "airtap": "^3.0.0",
+    "browserify": "^16.5.1",
     "standard": "^14.3.4",
-    "tape": "^5.0.0"
+    "tape": "^5.0.0",
+    "tinyify": "^3.0.0",
+    "uglify-js": "^3.10.1"
```
Comment (subins2000) [inline package.json:27]: Is this dependency necessary ? Only tinify is needed, right ?

File: package.json [hunk 4]
```diff
@@ -5,22 +5,26 @@
   "main": "node.js",
   "browser": "p2pt.js",
   "scripts": {
-    "build": "browserify ./p2pt.js -s P2PT > ./dist/p2pt.umd.js",
+    "build": "npm run build:umd && npm run build:umd-tiny",
+    "build:umd": "browserify ./p2pt.js -s P2PT > dist/p2pt.umd.js",
+    "build:umd-tiny": "browserify ./p2pt.js -p tinyify -s P2PT > dist/p2pt.umd.min.js",
```
Comment (subins2000) [inline package.json:10]: What does UMD stand for ? Wouldn't `p2pt.min.js` jus be easier ?

**Comment by therealadityashankar** [comment]
removed it umd is "universal module definition", see https://github.com/umdjs/umd , there are also other formats of release, I hope to eventually also release a built [esmodule](https://github.com/umdjs/umd), this is better to avoid leaking variables in the global scope, and I felt prefixing is better, for explicitness, when other build package get released

**Comment by subins2000** [comment]
TIL! Thank you for your contribution @therealadityashankar !

**Pull request merged**